app/components/Confirmation/Confirmation.js

```diff
@@ -50,20 +50,29 @@ class Confirmation extends Component {
     window.close();
   }
 
+  isEnabled() {
+    const article = this.props.article;
+    const isSaved = article.get('state');
+
+    const confirmationState = JSON.parse(localStorage.confirmation || '{}');
```

Comment:
Would be good to rename the whole thing to `duplicationConfirmation` as `confirmation` is too general.